feature-libs/customer-ticketing/core/connectors/customer-ticketing.connector.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { TestBed } from '@angular/core/testing';
+import { of } from 'rxjs';
+import { take } from 'rxjs/operators';
+import { CustomerTicketingAdapter } from './customer-ticketing.adapter';
+import { CustomerTicketingConnector } from './customer-ticketing.connector';
+
+import createSpy = jasmine.createSpy;
+
+class MockCustomerTicketingAdapter
+  implements Partial<CustomerTicketingAdapter>
+{
+  getTicket = createSpy().and.returnValue(of({}));
```

Comment:
the getTicket is supposed to return 'something'

let's assert that the subscribed value returns that 'something'